### PR TITLE
Update doc for wp language core install command. 

### PR DESCRIPTION
**Pull request by UmeshSingla**
Related: 
https://github.com/wp-cli/language-command/issues/90
https://github.com/wp-cli/language-command/issues/90#issuecomment-2148970327
https://github.com/wp-cli/wp-cli/issues/5955

### CODE REVIEW

**Comment by ernilambar** [comment]
Do we have official page with list of all WordPress Locale codes? 

https://translate.wordpress.org/ 
In this page also, it is not clear about Language Code and WordPress Locale code.

**Comment by ernilambar** [comment]
@swissspidy Context https://github.com/wp-cli/language-command/issues/90#issuecomment-2148970327

**Comment by swissspidy** [comment]
OK, thanks I see the context now on the issue.

Instead of adding the same example for multiple locales, it would be better to just explain the expected format, in case that it's unclear.

> Do we have official page with list of all WordPress Locale codes?
> In this page also, it is not clear about Language Code and WordPress Locale code.

It's the grey text below the name:

Screenshot 2024-06-13 at 10 52 37

File: README.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ Installs a given language.
 wp language core install <language>... [--activate]
 ~~~
 
-Downloads the language pack from WordPress.org.
+Downloads the language pack from WordPress.org. Find your language code at: https://translate.wordpress.org/
```
Comment (swissspidy) [inline README.md:153]: Please note that all the content in `README.md` is derived from the source code.

You should not edit this file directly, but only update it in the source code.

Comment (ernilambar) [reply README.md:153]: @UmeshSingla It should be updated here https://github.com/wp-cli/language-command/blob/main/src/Core_Language_Command.php#L171

**Comment by UmeshSingla** [comment]
@swissspidy 

> Instead of adding the same example for multiple locales, it would be better to just explain the expected format, in case that it's unclear.

While on it, should I update the existing example with a different language code, which has language code along with the country code in it?

File: src/Core_Language_Command.php [hunk 2]
```diff
@@ -168,26 +168,26 @@ public function is_installed( $args, $assoc_args = array() ) {
...
 	 * Installs a given language.
 	 *
-	 * Downloads the language pack from WordPress.org.
+	 * Downloads the language pack from WordPress.org. Find your language code at: https://translate.wordpress.org/
 	 *
 	 * ## OPTIONS
 	 *
 	 * <language>...
-	 * : Language code to install.
+	 * : Language code to install, it is based on the ISO-639 language code (e.g. pt for Portuguese) followed by the ISO-3166 country code (e.g. PT for Portugal or BR for Brazil). So, the Brazilian Portuguese file would be called pt_BR
```
Comment (ernilambar) [inline src/Core_Language_Command.php:176]: This explanation does accommodate the cases of same Language Code and Locale Code, `ja`, `ar`, etc.

Comment (swissspidy) [reply src/Core_Language_Command.php:176]: I think just linking to the website above is enough, no?

Comment (UmeshSingla) [reply src/Core_Language_Command.php:176]: I think it is enough, with the updated example and the url it should be fairly easy to identify a proper language code.

Comment (ernilambar) [reply src/Core_Language_Command.php:176]: Ok, then lets revert this explanation and add just link only in the description.

File: README.md (hunk 1 above)
Comment (ernilambar) [reply README.md]: @swissspidy In core handbook, for language code, `https://make.wordpress.org/polyglots/teams/` is given in some places. Which link is more suitable here, this one or this one `https://translate.wordpress.org/` ?

File: src/Core_Language_Command.php (hunk 2 above)
Comment (UmeshSingla) [reply src/Core_Language_Command.php]: Reverted it. Have kept the URL same, waiting for feedback.

**Review by swissspidy — APPROVED** [review]

**Pull request merged**